example/ios/Demo/Beacons/BeaconsViewCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  BeaconsViewCell.swift
+//  Demo
+//
+//  Created by Grigory Avdyushin on 05/08/2020.
+//  Copyright © 2020 Splendo. All rights reserved.
+//
```

Comment:
use the same copyright header as kotlin (does XCode have any settings for this?)

Reply:
Added template